src/services/contact/sendMessageService.tsx

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+import Endpoints from 'constants/endpoints';
+
+const sendMessageService = async (email: String, message: String) => {
+  const contactEndpoint = `${process.env.REACT_APP_API_BASE_URL}${Endpoints.SEND_QUESTION}`;
+  await axios.post(contactEndpoint, { email: email, message: message })
```

Comment:
Since the property name and the value are the same, you can omit being redundant by writing it like this:
```suggestion
  await axios.post(contactEndpoint, { email, message })
```